jupyter_kernel_mgmt/hl.py

```diff
@@ -0,0 +1,79 @@
+"""High-level APIs"""
```

Comment:
`hl.py` strikes me as a strange name.  Could this be changed to perhaps something along the lines of _execution_ or _helpers_ (e.g., `from .execution import run_kernel_blocking` or `from .helpers import run_kernel_blocking`)?

Reply:
I was borrowing hl for high-level from h5py. I'm exposing them all at the top level of the module (i.e. `from jupyter_kernel_mgmt import run_kernel_blocking`), so the internal module name shouldn't matter too much, but I'm happy to make it something like `helpers` if you prefer.

Reply:
Ah, I see.  That makes sense (and shows my naïveté in python :blush:).   I suppose the tests that reference these can't use that import because they need to run from the local files.   Is that correct?

Seems like we should document these four methods in a "High-level API" section or something of that sort.  I'd be okay differing that until #28 is merged or we ask @echarles to include these prior to #28's merge.

Reply:
No, it's fine to import them that way in the tests (I've made the change now). I was just putting it together in a hurry. That's also why I didn't get to docs, which should certainly exist.